apps/design/backend/schema.sql

```diff
@@ -16,3 +16,19 @@ create table background_tasks (
   completed_at timestamp,
   error text
 );
+
+create table translation_cache (
+  source_text text not null,
+  target_language_code text not null,
+  translated_text text not null
+);
+
+create unique index idx_translation_cache on translation_cache (
+  source_text,
+  target_language_code
+);
+
+create table speech_synthesis_cache (
+  source_text text primary key,
+  audio_clip_base64 text not null
+);
```

Comment:
I recognize that, while we're all running locally, these caches 1) won't be shared across Vx employees and 2) will frequently get cleared, e.g. every time we make a DB schema change or run `pnpm clean-all`, etc. I'm not too concerned about costs right now, but as a stopgap until we deploy VxDesign, I may later add some logic to "bootstrap" everyone's local cache.